frameit/frames_generator/Rakefile

```diff
@@ -124,6 +158,16 @@ def measure_slot(path)
   begin
     command_output = MiniMagick::Tool::Magick.new do |magick|
       magick << path
+
+      # some devices (e. g. Samsung Galaxy S9) may have shadows inside their screen to show the round edges - we need
```

Comment:
This (and following lines) looks like a possible reason for the changed height of iOS frames from https://github.com/fastlane/frameit-frames/pull/10#issuecomment-514535858. Could you test locally if removing this logic here adds that 1px back? If so, this here could possibly have negative effect on iOS (will have to look into the exact details on the iOS differences to identify if this is really what is happening.)

Reply:
These lines didn't change the height, because these are in the measure_slot function which only modifies the image in memory when searching the offsets which are subsequently saved to offsets.json. The code which modifies and saves the frames is in convert_image_resource. I still tested it according to your instructions and the generated size is still off by 1 PX. 

However, I found something suspicious. [In this pull request](https://github.com/fastlane/fastlane/pull/14199), a change in frames_generator has been introduced which should avoid these 1 - 2 PX differences. This PR was merged on February 20, but last commit in the frameit-frames repo was on January 19. Is it possible the frames were not generated with the new code yet and my commit is the first one which brings these fixed sizes to the frameit-frames repo?

Reply:
Yes, that makes total sense. Let's pull @mollyIV in here to confirm.

Reply:
I've just tried removing the "co.fuzz("90%")" line and generating frames. Apple iPad Air 2 Gold had 2496 px height instead of 2495 px. So that's it.